module-apps/apps-common/AppStartupReason.hpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
```

Comment:
Application ? Sometimes it is App and sometimes Application

Reply:
I would totally vote for naming unifivation, just in separate PR for all of these to know what naming shall be followed. This is in `app` namespace therefore I called it AppStartupReason, it can be StartupReason for me - it really doesn't matter to me.